### PR TITLE
feat(frontend/proposal_list): add content uploaded status

### LINKED ISSUE #242: add content upload status to proposals list page
jj

we can remove username from the page and can add content upload status, so it will be easy for reviewers

### PR DESCRIPTION
**Pull request by umulingu**
closes #242

### CODE REVIEW

**Comment by theskumar** [comment]
@umulingu possible to upload a screenshot for your change here, please?

File: junction/templates/proposals/list.html [hunk 1]
```diff
@@ -91,7 +91,7 @@
         </div>
 
         {% if proposals_to_review %}
-        {% include 'proposals/partials/proposal-list--items.html' with proposals=proposals_to_review title="Proposals To Be Reviewed" display_status=False %}
+        {% include 'proposals/partials/proposal-list--items.html' with proposals=proposals_to_review title="Proposals To Be Reviewed" display_status=False content_status=True %}
```
Comment (theskumar) [inline junction/templates/proposals/list.html]: This is always `content_status` is always going to be `true`, wondering why would you like to do something like this. You need implement actual logic for detecting content uploaded or not.

Comment (umulingu) [inline junction/templates/proposals/list.html]: Because we have to display content status for Proposal to review tab only.

Comment (umulingu) [inline junction/templates/proposals/list.html]: screenshot

**Comment by ChillarAnand** [comment]
Closing as  this is  fixed. Thank you!

**Pull request closed without merging**